ftw/zipextract/browser/zipextract_view.py

```diff
@@ -0,0 +1,63 @@
+from Products.Five.browser import BrowserView
+from ftw.zipextract.zipextracter import ZipExtracter
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from plone.i18n.normalizer import idnormalizer
+import os
+from z3c.form import form
+
+
+class ZipExtractView(form.Form):
```

Comment:
None indeed. I got confused at some point and misinterpreted your comment https://github.com/4teamwork/ftw.zipextract/pull/1#discussion_r160420777 and thought we wanted to avoid having to rebuild the `zipextracter` object in when doing the extraction. I tried to use buttons for that, to avoid having to use the `__call__` method when doing the extraction. I realize that was really foolish 👎. Anyway, bottom line is, I'm back to using the BrowserView (hence the unused import too...)